cpp/src/rolling/detail/range_window_bounds.hpp

```diff
@@ -98,7 +101,8 @@ void assert_non_negative([[maybe_unused]] T const& value)
 
 template <typename RangeT,
           typename RepT,
-          CUDF_ENABLE_IF(std::is_integral_v<RangeT> && !cudf::is_boolean<RangeT>())>
+          CUDF_ENABLE_IF(std::is_floating_point_v<RangeT> ||
+                         (std::is_integral_v<RangeT> && !cudf::is_boolean<RangeT>()))>
```

Comment:
```suggestion
          CUDF_ENABLE_IF(cudf::is_numeric<RangeT>() && !cudf::is_boolean<RangeT>())>
```
Could these also use the `cudf::is_numeric` instead? There are 3 other patterns similar to this in this file.

Reply:
Good idea. Fixed.